plugin/pkg/admission/limitranger/admission.go

```diff
@@ -68,7 +68,6 @@ type LimitRanger struct {
 
 var _ admission.MutationInterface = &LimitRanger{}
 var _ admission.ValidationInterface = &LimitRanger{}
-var _ kubeapiserveradmission.WantsInternalKubeInformerFactory = &LimitRanger{}
```

Comment:
it's duplicated with L207

Reply:
Would prefer to move all of those here, next to the type.